src/olympia/addons/views.py

```diff
@@ -859,3 +859,26 @@ def finalize_response(self, request, response, *args, **kwargs):
             request, response, *args, **kwargs)
         patch_cache_control(response, max_age=60 * 60 * 6)
         return response
+
+
+class LanguageToolsView(ListAPIView):
+    authentication_classes = []
+    pagination_class = None
+    permission_classes = []
+    serializer_class = LanguageToolsSerializer
+
+    def get_queryset(self):
+        try:
+            application_id = AddonAppFilterParam(self.request).get_value()
+        except ValueError:
+            raise ParseError('Invalid app parameter.')
+
+        types = (amo.ADDON_DICT, amo.ADDON_LPAPP)
+        return Addon.objects.public().filter(
+            appsupport__app=application_id, type__in=types,
+            target_locale__isnull=False).exclude(target_locale='')
+
+    def list(self, request, *args, **kwargs):
```

Comment:
This is not a straight re-implementing: I'm wrapping the data in a `{'results': data}` dict to mimic pagination behaviour